module.json

```diff
@@ -10,7 +10,13 @@
       "email": "arcanistzed@gmail.com",
       "discord": "arcanist#4317",
       "reddit": "u/arcanistzed"
-    }
+    },
+		{
+			"name": "Michael Leahy (TyphonJS)",
+			"url": "https://github.com/typhonrt",
+			"discord": "MLeahy#4299",
+			"reddit": "u/typhonrt"
+		}
```

Comment:
Indentation is incorrect

Reply:
Kind of pointless to tell me.... 

![image](https://user-images.githubusercontent.com/311473/179154159-625d8b46-bd29-447b-bc37-70806c7a0494.png)

I fixed things up according to your eslint file though I did comment out the very tail end regarding curly braces. You can certainly add back in that eslint rule and make the changes otherwise the codebase should be clean and reasonably commented.